src/server/rpc/procedures/waterwatch/waterwatch.js

```diff
@@ -0,0 +1,199 @@
+// TODO change console logs.
+// eslint
+let WaterWatchRPC = {
+    isStateless: true,
+    getPath: () => '/waterwatch'
+};
+
+let debug = require('debug'),
+    // request = require('request'),
+    rp = require('request-promise'), //maybe use request-promise-native?
+    error = debug('netsblox:rpc:trends:error'),
+    CacheManager = require('cache-manager'),
+    trace = debug('netsblox:rpc:trends:trace'),
+    Q = require('q');
+
+let baseUrl = 'https://waterservices.usgs.gov/nwis/iv/?',
+    cache = CacheManager.caching({store: 'memory', max: 1000, ttl: 36000});
+
+
+
+// let northernLat = 38;
+// let easternLong = -81;
+// let southernLat = 37;
+// let westernLong = -82;
+
+
+// turn an options object into query friendly string
+function encodeQueryData(options) {
+   let ret = [];
+   for (let d in options)
+     ret.push(encodeURIComponent(d) + '=' + encodeURIComponent(options[d]));
+   return ret.join('&');
+}
+
+// used to send feedback about errors to the caller.
+function showError(err, response) {
```

Comment:
this function is used in one spot - why don't you just put these two lines where this is currently called? That is, abstracting this logic into a function doesn't seem to be gaining anything